sym/manifest.json

```
@@ -0,0 +1,64 @@
+{
+    "manifest_version": "2.0.0",
+    "app_uuid": "d81d1dd3-d5e8-4373-98a6-f95b797b3f9d",
+    "app_id": "sym",
+    "display_on_public_website": true,
+    "tile": {
+        "overview": "README.md#Overview",
+        "configuration": "README.md#Setup",
+        "support": "README.md#Support",
+        "changelog": "CHANGELOG.md",
+        "description": "Send Sym Audit Logs to Datadog",
+        "title": "Sym",
+        "media": [
+            {
+                "media_type": "video",
+                "caption": "A Sym Overview Video",
+                "image_url": "images/sym_video_thumbnail.jpg",
+                "vimeo_id": 123456789
+            },
+            {
+                "media_type": "image",
+                "caption": "Sym helps you build access and approval workflows using your native platform engineering tools",
+                "image_url": "images/home_hero_image.png"
+            },
+            {
+                "media_type": "image",
+                "caption": "Define and deploy access rules in Terraform",
+                "image_url": "images/define_deploy.jpg"
+            },
+            {
+                "media_type": "image",
+                "caption": "Request, approve, and deny anywhere in Slack",
+                "image_url": "images/request_approve.jpg"
+            }
+        ],
+        "classifier_tags": [
+            "Submitted Data Type::Logs"
+        ]
+    },
+    "assets": {
+        "dashboards": {
+            "Sym Overview": "assets/dashboards/sym_overview.json"
+        },
+        "integration": {
+            "source_type_name": "sym",
+            "configuration": {},
+            "events": {
+                "creates_events": false
+            },
+            "service_checks": {
```

```suggestion
            },
            "metrics": {
             "metadata_path": "metadata.csv",
             "check": [],
             "prefix": "sym."
           },
            "service_checks": {
```

Even if the integration is not sending in metrics, the metrics metadata.csv file with the pre-populated headers is still required.

Can you please add the metadata.csv file back? [https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#complete-the-necessary-integration-asset-files](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#complete-the-necessary-integration-asset-files)